packages/connections/src/components/connection-list/connection-list.tsx

```diff
@@ -73,6 +76,28 @@ function ConnectionList({
   createNewConnection: () => void;
   setActiveConnectionId: (connectionId?: string) => void;
 }): React.ReactElement {
+  const favoriteConnections = connections
+    .filter(
+      (connectionInfo): connectionInfo is ConnectionInfoFavorite =>
+        !!connectionInfo.favorite
+    )
+    .sort((a: ConnectionInfoFavorite, b: ConnectionInfoFavorite) => {
+      return `${b.favorite.name}`.toLowerCase() <
+        `${a.favorite.name}`.toLowerCase()
```

Comment:
Can `a.favorite.name` or `b.favorite.name` be something other than strings? The TS definitions don’t seem to imply so :)

Also, probably won’t matter a lot, but since this is user-facing, maybe `toLocaleLowerCase()` is the better choice?

Reply:
You can also use [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) here so you don't need to handle lowercasing stuff and return value yourself